### ISSUE TITLE
Feature: Make /clubs more accessible

### ISSUE THREAD

**Issue by Olaleye-Blessing** [MilanCommunity/Milan#1105]
### Description

The [clubs' page](https://milanhub.org/clubs) lacks some accessibility as shown in the screenshots cause of the wrong use of elements. Using the right elements will greatly improve its accessibility.

### Screenshots

<img width="1464" alt="Screenshot 2023-10-01 at 21 38 28" src="https://github.com/MilanCommunity/Milan/assets/70102539/5f97a6ca-efd7-4187-84b1-b9d4b25a6b2b">

### 🥦 Browser

Brave

### Checklist ✅

- [X] I checked and didn't find similar [issue](https://github.com/MilanCommunity/Milan/issues)
- [X] I have read the [Contributing Guidelines](https://github.com/MilanCommunity/Milan/blob/main/CONTRIBUTING.md)
- [X] I am willing to work on this issue (blank for no)

**Comment by tamalCodes** [comment]
Hi @Olaleye-Blessing , thanks for pointing that out. Can you perhaps explain what more there is to fix this issue ?
From what I can see there is a use of the `h1` tag. What do you suggest to fix this issue ?

**Comment by Olaleye-Blessing** [comment]
Ideally, there should be only 1 `h1` per page which is the header: `Caring Hands, Changing Futures.`. So the `h1` I showed in the previous screenshot should probably be `h2`.

Also, those clubs are a list of clubs, so `ul` > `li` is appropriate in this situation. 

There should be a main element on the page, a header too.

**Comment by tamalCodes** [comment]
That sounds good. You can start working on the issue, assigning it to you 🚀 
Make sure that the styles are intact across all the devices when the changes are made to the tags. 
<br/>

## 📑 Checklist in case you missed something

- [Local project setup, .env](https://github.com/MilanCommunity/Milan#1%EF%B8%8F-setting-up-the-project-locally) for the frontend.
- [Contributing guidelines](https://github.com/MilanCommunity/Milan#2%EF%B8%8F-contributing-guidelines--more) to be followed.


Please maintain structure, use conventional pull request titles, and avoid spam to avoid closure or bans.
